components/compliance-service/reporting/relaxting/migrations_from_a1.go

```diff
@@ -20,6 +20,10 @@ type A1ElasticSearchIndices struct {
 	backend *ES2Backend
 }
 
+func (migratable A1ElasticSearchIndices) migrateCompRunInfo() error {
```

Comment:
we should not touch these migrations, these are for time series migrations, we might need to write single migration file